lib/improver/tests/utilities/cube_manipulation/test_merge_cubes.py

```diff
@@ -120,18 +137,71 @@ def test_lagged_ukv(self):
     def test_multi_model(self):
         """Test Multi models merge OK"""
         cubes = iris.cube.CubeList([self.cube, self.cube_ukv])
-        result = merge_cubes(cubes)
+        result = merge_cubes(cubes, "mosg__model_configuration")
         self.assertIsInstance(result, Cube)
         self.assertArrayAlmostEqual(
-            result.coord("model_realization").points, [3000., 4000.,
-                                                       4001., 4002.])
+            result.coord("model_realization").points, [0., 1.,
+                                                       2., 1000.])
 
     def test_threshold_data(self):
         """Test threshold data merges OK"""
         cubes = iris.cube.CubeList([self.prob_ukv, self.prob_enuk])
-        result = merge_cubes(cubes)
+        result = merge_cubes(cubes, "mosg__model_configuration")
+        self.assertArrayAlmostEqual(
+            result.coord("model_id").points, [0., 1000.])
+
+    def test_non_mo_model_id(self):
+        """Test that a model ID attribute string can be specified when
+        merging multi model cubes"""
+        cubes = iris.cube.CubeList(
+            [self.cube_non_mo_ens, self.cube_non_mo_det])
+        result = merge_cubes(cubes, 'non_mo_model_config')
+        self.assertIsInstance(result, Cube)
         self.assertArrayAlmostEqual(
-            result.coord("model_id").points, [3000., 4000.])
+            result.coord(
+                "model_realization").points, [0., 3., 4., 1000., 1001., 1002.])
```

Comment:
Is this the behaviour we expect?  Why are two realizations created out of nowhere?